timemachine/ff/handlers/nonbonded.py

```diff
@@ -159,23 +162,22 @@ def compute_or_load_am1_charges(mol):
     """Unless already cached in mol's "AM1Cache" property, use OpenEye to compute AM1 partial charges."""
 
     # check for cache
-    cache_key = "AM1Cache"
-    if not mol.HasProp(cache_key):
+    if not mol.HasProp(AM1_CHARGE_CACHE):
         # The charges returned by OEQuacPac is not deterministic across OS platforms. It is known
         # to be an issue that the atom ordering modifies the return values as well. A follow up
         # with OpenEye is in order
         # https://github.com/openforcefield/openff-toolkit/issues/983
         am1_charges = list(oe_assign_charges(mol, "AM1"))
 
-        mol.SetProp(cache_key, base64.b64encode(pickle.dumps(am1_charges)))
+        mol.SetProp(AM1_CHARGE_CACHE, base64.b64encode(pickle.dumps(am1_charges)))
 
     else:
-        am1_charges = pickle.loads(base64.b64decode(mol.GetProp(cache_key)))
+        am1_charges = pickle.loads(base64.b64decode(mol.GetProp(AM1_CHARGE_CACHE)))
 
     return np.array(am1_charges)
 
 
-def bond_smirks_matches(mol, smirks_list):
+def compute_or_load_bond_smirks_matches(mol, smirks_list):
```

Comment:
Nit: updates behavior but does not update docstring

Reply:
Fixed in [21039c3](https://github.com/proteneer/timemachine/pull/640/commits/21039c3556e0ba96419309922e6b5c41f8411330)